scripts/src/primazatest/options/primaza-alice.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: primaza.io/v1alpha1
-kind: Tenant
-manifestDirectory: ./out/config
-name: primaza-alice
-version: latest
-controlPlane:
-  context: kind-primazactl-tenant-test
-  internalUrl: null
-  kubeconfig: ~/.kube/config
 clusterEnvironments:
-- name: worker-alice
+- applicationNamespaces:
+  - name: alice-app
   environment: test
+  name: worker-alice
+  serviceAccountNamespace: worker-sa
+  serviceNamespaces:
+  - name: alice-svc
   targetCluster:
     context: kind-primazactl-join-test
     internalUrl: https://172.18.0.3:6443
     kubeconfig: ~/.kube/config
-  serviceNamespaces:
-    - name: alice-svc
-  applicationNamespaces:
-    - name: alice-app
-
+controlPlane:
+  context: kind-primazactl-tenant-test
+  internalUrl: https://172.18.0.2:6443
```

Comment:
this can introduce some problems, should we revert this changes?

Reply:
will do